File: sdk-python/src/c6_guardrails/types/health.py
from dataclasses import dataclass
from typing import Literal


@dataclass
class GatewayHealth:
    status: Literal["healthy", "unhealthy"]
    timestamp: str
    data_db: bool
    log_db: bool
    cache_loaded_at: str | None
    cache_next_reload_at: str | None
    cache_next_reload_in: str | None
    detection_degraded: bool

File: sdk-node/src/types/health.ts
export interface GatewayHealth {
  status: 'healthy' | 'unhealthy';
  timestamp: string;
  data_db: boolean;
  log_db: boolean;
  cache_loaded_at: string | null;
  cache_next_reload_at: string | null;
  cache_next_reload_in: string | null;
  detection_degraded: boolean;
}
